src/server.js

```diff
@@ -27,6 +30,7 @@ router.post('/shelters', sheltersController.postShelters);
 router.get('/shelters/:shelterID', sheltersController.getShelter);
 router.delete('/shelters/:shelterID', sheltersController.deleteShelter);
 router.patch('/shelters/:shelterID', sheltersController.updateShelter);
+router.get('/shelters/:shelterID/pets', sheltersController.getPets);
```

Comment:
Just a side note, the getShelters route is still /shelters/test, do we want to remove the /test?

Reply:
Removed the test since we are not using it.